src/vdaf/prg.rs

```diff
@@ -48,12 +52,24 @@ impl<const SEED_SIZE: usize> Seed<SEED_SIZE> {
     }
 }
 
+impl<const SEED_SIZE: usize> Default for Seed<SEED_SIZE> {
```

Comment:
I'm worried that this is too easy to use wrong. You mention in the PR description that we expect clients to take `Seed::default` and then fill the inner buf with random bytes, but nothing stops them from working with the default seed.

We already have `Seed::from_bytes` that lets callers create a seed with random bytes they provide. Is that not good enough? Is the concern that it forces them to somehow instantiate a `[u8; SEED_SIZE]` and move it in? If so, we could provide `pub from_byte_iter<I = Iterator<u8>>(seed_bytes: I) -> Self` which would read `SEED_SIZE` bytes from the iterator (maybe we could refactor the existing `from_bytes` into something that takes `IntoIterator<Item = u8>`). Is that flexible enough?

Reply:
Another way around this would be to make the SeedableRng::Seed associated type just `[u8; 16]`. (more in line with other implementations) That'll let us avoid any changes to our `Seed` struct.